### ISSUE TITLE
nv_peer_mem dkms fails on ubuntu 18.04,

### ISSUE THREAD

**Issue by JohnSpillerNvidia** [Mellanox/nv_peer_memory#32]
I'm not sure if this is an issue with nv_peer_mem or with the something else, but trying to build the nv_peer_mem  module against the 4.15.0-10 kernel in Ubuntu bionic fails because ACCESS_ONCE is undefined.   ACCESS_ONCE was removed from compiler.h in the kernel  in december 2017.

**Comment by ferasd** [comment]
You are right, the macro was removed from newer kernels. We may convert ACCESS_ONCE() to READ_ONCE(), I will create a pull request for that.

**Comment by ferasd** [comment]
@JohnSpillerNvidia can we close?

**Comment by JohnSpillerNvidia** [comment]
?I think so.  I just got back from vacation and haven't tried your build, but I did exactly the same thing and it compiled before I left, so we should be fine.

________________________________
